admin.md

```diff
@@ -115,40 +148,58 @@ The `GetTenant` endpoint can also take in a `tenantId` and will return a list of
+
+**Example** 
+
+With the tenant ID `24` and Data Integrator ID `producer`, the resulting two topics are `ingest.24.producer.in` and `ingest.24.producer.notification`. In addition, the tenant ID can include a `tenant` prefix, for example, `ingest.tenant24.producer.in` and `ingest.tenant24.producer.notification`. However, it must be consistent with the Elasticsearch [index tenant id](#create-a-new-tenant-in-elasticsearch). 
+
+**Note:** Only use lowercase alphanumeric characters, hyphen (-), and underscore (\_\).
 
 ### Using the Management API stream endpoints
 
-Setting up HRI for a new Data Integrator also requires creating a [Stream](glossary.md#stream). The Management API contains three `Stream` endpoints to help you manage Streams: `Create`, `Get`, and `Delete`. Please note that all three of these endpoints require an API key and IAM Authentication - you will need to pass in an IAM Bearer token as part of the Authorization header in the requests. 
+Setting up Health Record Ingestion for a new Data Integrator also requires creating a [stream](glossary.md#stream). The Management API contains three `Stream` endpoints to help you manage the streams: `Create`, `Get`, and `Delete`. 
 
-In the case of `Create` and `Delete`, the IAM bearer token must be associated with a user who has Manager role permissions. For `Get`, the bearer token must be associated with a user who has at least Reader role permissions. See [Event Streams documentation](https://cloud.ibm.com/docs/EventStreams?topic=EventStreams-security#assign_access) for details on permissions.
+**Note:** Each of these endpoints requires an API key and IAM Authentication.  In requests, you must pass an IAM Bearer token in the Authorization header. 
 
-#### Create Stream
-You can use the Management API Create Stream endpoint to create the `in` and `notification` topics for the Tenant and Data Integrator pairing.
+With `Create` and `Delete`, the IAM bearer token must be associated with a user who has Manager role permissions. For `Get`, the bearer token must be associated with a user who has at least Reader role permissions. For more information on permissions, see the [Event Streams documentation](https://cloud.ibm.com/docs/EventStreams?topic=EventStreams-security#assign_access).
 
-The `Create` Stream endpoint takes in two path parameters, `tenantId` and `streamId`, where `streamId` is made up of the Data Integrator Id and an optional qualifier, delimited by '.'. 
+#### Create Stream endpoint
 
-Both `tenantId` and `streamdId` **may only contain lowercase alpha numeric characters, -, and _**. `streamdId` may also contain one '.'. For example, for the `tenantId` "tenant24", Data Integrator Id "data-int-1" and optional qualifier "qualifier1", you could use the following curl command:
+For the tenant and Data Integrator pairing, you can use the Management API Create Stream endpoint to create the `in` and `notification` topics.
```

Comment:
Let's get rid of 'For the tenant and Data Integrator pairing'.  How about just "You can use..."

Reply:
Done